src/modules/feature/selectors.ts

```diff
@@ -0,0 +1,71 @@
+import { LoadingState } from '../loading/reducer'
+import { FeatureState } from './reducer'
+import { Applications, Feature } from './types'
+
+export const getState = (state: any): FeatureState => state.feature!
+export const getData = (state: any): Record<Applications, Feature> =>
+  getState(state).data
+export const getLoading = (state: any): LoadingState => getState(state).loading
+export const getError = (state: any): string | null => getState(state).error
+
+/**
+ * Helper to get if a feature flag is whether enabled or disabled.
+ * It will first look into your env file for the feature flag, if it is not defined there,
+ * it will look it in the requested and stored features data.
+ * The env key will be determined from the application and the flag. For example, if the
+ * application is "explorer" and the flag is "some-crazy-feature", it will look
+ * for it as REACT_APP_EXPLORER_SOME_CRAZY_FEATURE.
```

Comment:
After having FF in all dapps, do we still want to have environment variables as FF?

Reply:
I think this is helpful for local development